src/cloudsc_pyiface/pyproject.toml

```diff
@@ -23,10 +23,10 @@ classifiers = [
    " Topic :: Scientific/Engineering :: Atmospheric Science "
 ]
 dependencies = [
+    "numpy",
```

Comment:
I believe we should let f90wrap choose its own numpy version and thus not specify  numpy here. Newest lumpy doesn't support f90wrap AFAIK.

Reply:
We do not prescribe a version for exactly this reason, thus leaving it to pip to find an appropriate version that satisfies the respective constraints. However, our own utility routines have an explicit dependency on numpy (`cloudsc_data.py` does `import numpy as np`) and this version free dependency should be explicitly stated.